monitor/utils/api.py

```diff
@@ -49,7 +49,7 @@ def patch(self, rule, status_code=202):
         return self._mroute('PATCH', rule, status_code)
 
     def _mroute(self, methods, rule, status_code=None, **kw):
-        if type(methods) is str:
```

Comment:
What is the main purpose of this modification? Can you please document this refactor in the description of this PR?